keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -487,6 +488,24 @@ free to use non-namespaced properties (e.g. `fingerprint`) as they see fit, but
 any shared tooling should use appropriately namespaced names.
 
 
+#### Property Ladder
+
+We foresee that there will be more properties that the community wants to adopt as "well known" properties. Thus, we want to define a process for adopting a property to be a well known property in the future:
+
+1. **Extension Property Stage**: A property is implemented as an extension property if the implementer has no intent to elevate it to a standard property. This property cannot use the reserved suffixes and is not required to be implemented by all implementers.
+
+2. **Standard Property Stage**: The implementor can propose to add a property to be a **standard** property which means if anyone implements the property with the same name, it must follow the same definition and rules described in the KEP. Not all implementers are required to implement the property.
+   The process for proposing a standard property is as follows:
+    * The property sponsor must present a discussion agenda with clear use cases and motivation at a Multi-cluster SIG meeting.
+    * The community will vote on the proposal, and if it receives approval from at least 2/3 of voters, the property becomes a standard property and is incorporated into the About API KEP.
```

Comment:
How are people eligible to vote? Do we need a quorum? Is there a cool off period between votes on similar proposals? Is there any veto powers by the sig co chairs? Is there anything from CNCF or other SIGs we can, must, or even want to draw on for regulations or tooling?

What, if any, are the obligations of the About API subproject or the proposer to facilitate implementation of a new standard property? For example do we ship the About API with as much webhook coverage as possible for the validation? Do we publish and/or run periodic About API conformance tests for standard properties, or only core properties, or none? If by conformance tests or otherwise we want to try to enforce compliance, how do we determine which implementations to "watch"?

Reply:
Cloud-event used to have a voting mechanism like this and I would propose something like this
1. Who are eligible to vote:  Every entity whose member collectively kept good attendence ( attend the meeting at least 4 out of the last 5 times) have one vote.
2. 2/3 of all eligible voters vote yes pass the proposal
3. The cool off period is 3 months
4. No veto power